azure-pipelines.yml

```diff
@@ -34,17 +34,10 @@ jobs:
   pool:
     vmImage: 'macOS-10.14'
   steps:
-  - task: UseDotNet@2
-    displayName: 'Use .NET Core SDK 3.1.101'
+  - task: DotNetCoreInstaller@0
+    displayName: 'Use .NET Core SDK 3.1.302'
```

Comment:
We were previously stuck on 3.1.101 due to a [problem with sharpgentools](https://github.com/SharpGenTools/SharpGenTools/issues/164). Given that it looks like that must no longer be an issue (due to the build now passing) would it be worth upgrading to the latest 3.1 patch (3.1.401 it looks like?)

Reply:
Seems like `3.1.401` is now deployed. Probably worth bumping the version now 🎉
https://github.com/actions/virtual-environments/issues/1407#issuecomment-681107768

Reply:
Done! Wondering if we need to adjust some additional stuff to ship `Avalonia.ReactiveUI.Events` to NuGet.